src/overlays/actors/ovl_Obj_Hana/z_obj_hana.c

```diff
@@ -27,11 +26,74 @@ const ActorInit Obj_Hana_InitVars = {
     (ActorFunc)ObjHana_Update,
     (ActorFunc)ObjHana_Draw,
 };
-*/
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_Obj_Hana/ObjHana_Init.s")
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_Obj_Hana/ObjHana_Destroy.s")
+static ColliderCylinderInit sCylinderInit = {
+    { COLTYPE_UNK10, 0x00, 0x00, 0x39, 0x20, COLSHAPE_CYLINDER },
+    { 0x00, { 0x00000000, 0x00, 0x00 }, { 0x00000000, 0x00, 0x00 }, 0x00, 0x00, 0x01 },
+    { 8, 10, 0, { 0, 0, 0 } },
+};
+
+static CollisionCheckInfoInit sColChkInfoInit = { 0, 0xC, 0x3C, 0xFF };
+
+typedef struct {
+    /* 0x00 */ Gfx* dList;
+    /* 0x04 */ f32 scale;
+    /* 0x08 */ f32 yOffset;
+    /* 0x0C */ s16 radius;
+    /* 0x0E */ s16 height;
+} HanaParams; // size 0x10
+
+static HanaParams hanaParams[] = {
```

Comment:
Our naming scheme is that all static variables (as in local to the file) should be prefixed with `s`, while global variables are prefixed with `g`.
```suggestion
static HanaParams sHanaParams[] = {
```